pl_bolts/callbacks/byol_updates.py

```diff
@@ -36,7 +39,8 @@ def __init__(self, initial_tau: float = 0.996):
         self.initial_tau = initial_tau
         self.current_tau = initial_tau
 
-    def on_train_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, dataloader_idx):
+    def on_train_batch_end(self, trainer: Trainer, pl_module: LightningModule, outputs: Sequence,
```

Comment:
lest spit all these multi-line to argument per line
```suggestion
    def on_train_batch_end(self, trainer: Trainer, pl_module: LightningModule, outputs: Sequence,
```